book/content/reviewing/03/typical_workflows.md

```diff
@@ -20,7 +20,7 @@ Before requesting a review, be sure you've met all the obvious quality benchmark
 - you have created [documentation](#Documentation) to the required standards of the project,
```

Comment:
"documentation of the required standards" or "documentation about the required standards"